History.md

```diff
@@ -115,8 +115,8 @@ commit logs available on GitHub](https://github.com/chaijs/chai/commits/master).
  * Refactor if statement out
  * Small unit test fix
  * Handle array indexing terminating paths
- * Merge pull request #309 from ericdouglas/iterableEqual-couting-once
- * couting variables just once
+ * Merge pull request #309 from ericdouglas/iterableEqual-couting -once
+ * couting  variables just once
```

Comment:
Unintentional?

Reply:
I was trying to change `couting ` to `counting `, but apparently lost two characters to make that happen :-o

`cout` is a thing, but I'm prety sure that this code was talking about counting...